packages/dialect-react-ui/components/Chat/screens/Main/ThreadsList/MessagePreview.tsx

```diff
@@ -64,7 +60,16 @@ export default function MessagePreview({
       </div>
       <div className="dt-flex dt-grow dt-border-b dt-border-neutral-600 dt-justify-between dt-truncate dt-pr-2">
         <div className="dt-flex dt-flex-col dt-max-w-full dt-truncate">
-          {dialect?.dialect.members.length > 0 && <div>{otherMemberStr}</div>}
+          {dialect?.dialect.members.length > 0 &&
+            (program?.provider.connection ? (
+              <CardinalDisplayAddress
+                connection={program?.provider.connection}
+                publicKey={otherMembers[0].publicKey}
+                showTwitterIcon={true}
```

Comment:
I guess, here should be some condition. Right no
<img width="1552" alt="Screenshot 2022-04-14 at 11 40 43" src="https://user-images.githubusercontent.com/2504771/163337815-74458fab-93f7-4041-8003-ea59d540d94a.png">
w the twitter icon showed even if there is no cardinal identity

Reply:
hey nice catch! added conditional rendering for this